docs/how_to/gpu_aware_mpi.md

```diff
@@ -75,6 +75,13 @@ make -j $(nproc)
 make -j $(nproc) install
 ```
 
+The following table shows the compatibility of UCX versions with ROCm versions.
+
```

Comment:
Can a link to the compatibility information be placed here instead?

Reply:
Actually, the information is at the moment only available in a Confluence page, so I am not sure where to add it other wise

Reply:
https://rocmdocs.amd.com/en/develop/release/3rd_party_support_matrix.html#communication-libraries

Reply:
The information is already out.

Reply:
done

Reply:
I will change the link to relative.